internal/handlers/handlers.go

```diff
@@ -137,3 +139,20 @@ func HandleGetJSONBlockByHeight(fetcher ifaces.IBlockFetcher) func(w http.Respon
 		io.WriteString(w, string(marshalledBlock))
 	}
 }
+
+// GetBlockFromResponse will convert the body of a reponse handler and return a Block
```

Comment:
converts the body of a response* , I don't know what a response handler is

Reply:
- [ ]